packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -70,12 +70,12 @@ class ApplicationCommand extends Base {
       this.name = data.name;
     }
 
-    if ('name_localizations' in data) {
+    if ('nameLocalizations' in data) {
       /**
        * The name localizations for this command
        * @type {?Object<Locale, string>}
        */
-      this.nameLocalizations = data.name_localizations;
+      this.nameLocalizations = data.nameLocalizations;
```

Comment:
The Discord API data comes with camel-cased keys

Reply:
This is true, I apologize.
But when you compare line 78 with line 88 you can see that the `this.nameLocalized` is also in correct camel case even when getting the non camel case value `data.name_localized`